libraries/Microsoft.Bot.Builder.Azure/CosmosDbPartitionedStorage.cs

```diff
@@ -378,6 +400,44 @@ private async Task CreateContainerIfNotExistsAsync()
             _container = containerResponse.Container;
         }
 
+        private string GetNestingErrorMessage(JObject json)
```

Comment:
Consider renaming to IsNestingError

Reply:
`IsNestingError` sounds like it should return a `bool` rather than a `string`. Should we still be returning a string even with the name `IsNestingError`, or should we refactor the code somehow?